docs/swagger.yaml

```diff
@@ -1711,6 +1728,96 @@ paths:
       summary: Login
       tags:
       - User
+  /api/permission:
```

Comment:
According to the zalando guidelines this should be pluralized: https://opensource.zalando.com/restful-api-guidelines/#134